README.md

```diff
@@ -22,23 +22,23 @@ resources in these repositories are going to be deployed in `monitoring`
 namespace in your Kubernetes cluster.
 
 - [prometheus-operator](katalog/prometheus-operator): Operator to deploy and
-  manage Prometheus and related resources.
+  manage Prometheus and related resources. Version: **0.29.0**
 - [prometheus-operated](katalog/prometheus-operated): Prometheus instance
   deployed with Prometheus Operator's CRD.
 - [alertmanager-operated](katalog/alertmanager-operated): Alertmanager instance
```

Comment:
I would also add the Alertmanager version deployed through the `Alertmanager` CRD.

Reply:
Ouch!!! Yeah, i forgot